R/status.R

```diff
@@ -86,6 +86,8 @@ the$status_running <- FALSE
 #'   cache are installed at the expected + proper locations, and validate the
 #'   hashes used for those storage locations.
 #'
+#' @param dev Boolean; should development dependencies be considered?
```

Comment:
You should only need to define this in one place, and then rely on `@inheritParams` to pull it in.

Reply:
I used `@inheritParams dependencies` for documenting the `dev` argument in both status.R and snapshot.R.
Thanks for the hint